sdk/eventhubs/azure-messaging-eventhubs-stress/src/main/java/com/azure/messaging/eventhubs/stress/scenarios/EventSenderBuffered.java

```diff
@@ -0,0 +1,73 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.messaging.eventhubs.stress.scenarios;
+
+import com.azure.core.util.logging.ClientLogger;
+import com.azure.messaging.eventhubs.EventData;
+import com.azure.messaging.eventhubs.EventHubBufferedProducerAsyncClient;
+import com.azure.messaging.eventhubs.EventHubBufferedProducerClientBuilder;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+import reactor.core.publisher.Flux;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+import java.util.stream.IntStream;
+import java.util.stream.Stream;
+import java.util.stream.StreamSupport;
+
+@Service("EventSenderBuffered")
+public class EventSenderBuffered extends EventHubsScenario {
```

Comment:
Do we have some criteria, on whether this scenario works good in stress test, or is broken in stress test?

Reply:
Currently I am working at testing, use different parameters (container configuration, number of events, etc), and check logs (errors) and monitor related metrics. Will summarize some document and base on that update stress test and sdk code.